api/db/changelog/db.changelog-223-modify-egress-bypass-window-datatype.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="yonghao" id="db.changelog-223-modify-egress-bypass-window-datatype">
+    <modifyDataType
+      columnName="description"
+      newDataType="varchar(8128)"
```

Comment:
whats the significance of 8128?

Reply:
Seems the access reason will be large text and we want to store that in our DB. So increase to 8128. Should be good for now

Reply:
I learnt this from this PR comment: https://github.com/DataBiosphere/leonardo/pull/3293#discussion_r1170539130

Reply:
I just realized in UI the MAX field the description field is 4000 https://github.com/all-of-us/workbench/blob/050b625bb0f9565f021090bb640333cff26b60b5/ui/src/app/pages/admin/user/admin-user-egress-bypass.tsx#L20C1-L20C37
, do we want to change that as well?

Reply:
Yes we need to keep these 2 values in agreement.

@yonghaoy is 4000 large enough?  It's not clear why 8128 was chosen here.

Reply:
ah. okay. I was thinking backend usually has a larger limit than FE. But 4000 also SG. Changed to 4000.